tests/test_auxtel_slew_and_take_image_checkout.py

```diff
@@ -42,6 +42,9 @@ def setUp(self) -> None:
         self.ataos_correction_status = True
         self.ataos_correction_result = "enabled"
         self.cmd_time = time.time()
+        self.m3_port_selected = 2
```

Comment:
So the way to do this is not to hard code it but to import the values from here:
https://github.com/lsst-ts/ts_idl/blob/develop/python/lsst/ts/idl/enums/ATMCS.py#L40

Then use variables names (e.g. M3State.Nasmyth2), same goes for the state.

Reply:
Ah great, thanks for the pointer. I'll add those changes now.